docs/csharp/programming-guide/main-and-command-args/top-level-statements.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Top-level statements - C# Programming Guide"
+description: Learn about top-level statements in C# 9 and later.
+ms.date: 03/08/2021
+helpviewer_keywords:
+  - "C# language, top-level statements"
+  - "C# language, Main method"
+---
+# Top-level statements (C# Programming Guide)
+
+Starting in C# 9, you don't have to explicitly include a `Main` method in a console application project. Instead, you can use the *top-level statements* feature to minimize the code you have to write. In this case, the compiler provides an implicit class and `Main` method entry point for the application.
```

Comment:
I personally lean towards using main without backticks instead of `Main`. Because the method the compiler generates isn't called `Main`, but `<Main>$`.

Reply:
I think "implicit" clarifies the status of `Main`, but in this instance it will be clearer to repeat "implicit" directly before Main.

Reply:
I don't think "implicit" resolves this. It could be implicit and called `Main`, but the compiler implementation decided it's called `<Main>$`.

And actually, from a compiler-point of view, it's not implicit at all (see discussion at https://github.com/dotnet/roslyn/pull/48199), but it's okay to mention implicit for the purpose of this article I think?